index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- font CDN -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- HEADER SECTION -->
+    <div class="header">
+        <div class="header__logo">David</div>
+        <div class="header__menu-btn">
+            <div class="menu-btn__ham"></div>
+        </div>
+    </div>
```

Comment:
- [ ] Consider wrapping this toolbar/header with a `header` tag instead of a `div` to improve the [semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) structure of your HTML.